### PR TITLE
Morpheus docs update post compartmentalization

### PR DESCRIPTION
**Pull request by AnuradhaKaruppiah**
- [x] Add a page for using the Morpheus conda packages
- [x] Add documentation for LLM and DFP stages APIs
- [] Update example READMEs (if needed)

### CODE REVIEW

File: docs/source/conda_packages.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+  conda config --env --add channels nvidia &&\
+  conda config --env --add channels rapidsai &&\
+  conda config --env --add channels pytorch
+```
+### Install the `morpheus-dfp` library
+```bash
+conda install -c nvidia morpheus-dfp
+```
+The `morpheus-dfp` Conda package installs the `morpheus_dfp` python package. It also pulls down all the necessary Conda runtime dependencies including [`morpheus-core`](https://anaconda.org/nvidia/morpheus-core).
+### Install additional `Pypi` dependencies
```
Comment (dagardner-nv) [inline docs/source/conda_packages.md:83]:
```suggestion
### Install additional PyPI dependencies
```

File: docs/source/conda_packages.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+Some of the DFP stages in the library require additional dependencies that are hosted on `Pypi`. These dependencies are included as a requirements file in the `morpheus_dfp` python package. And can be installed by running the following command:
```
Comment (dagardner-nv) [inline docs/source/conda_packages.md:84]:
```suggestion
Some of the DFP stages in the library require additional dependencies that are hosted on PyPI. These dependencies are included as a requirements file in the `morpheus_dfp` python package. And can be installed by running the following command:
```

File: docs/source/conda_packages.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+```bash
+python3 <<EOF
```
Comment (dagardner-nv) [inline docs/source/conda_packages.md:86]: Same as above, possibly simpler as a bash script

File: docs/source/conda_packages.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+import importlib.resources
+import subprocess
+requirements_file = importlib.resources.path("morpheus_dfp", "requirements_morpheus_dfp.txt")
+subprocess.call(f"pip install -r {requirements_file}".split())
+EOF
+```
+
+## Morpheus LLM
+The `morpheus-llm` library contains stages for Large Language Models (LLM) and  Vector Databases. These stages are used for setting up Retrieval Augmented Generation (RAG) pipelines. The `morpheus-llm` library is built from the source code in the `python/morpheus_llm` directory of the Morpheus repository.
```
Comment (dagardner-nv) [inline docs/source/conda_packages.md:95]:
```suggestion
The `morpheus-llm` library contains stages for interacting with Large Language Models (LLM) and Vector Databases (VDB). These stages are used for setting up Retrieval Augmented Generation (RAG) pipelines. The `morpheus-llm` library is built from the source code in the `python/morpheus_llm` directory of the Morpheus repository.
```

**Review by dagardner-nv — APPROVED** [review]
A few small things, overall looks good.

File: docs/source/conda_packages.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+The `morpheus-core` Conda package installs the `morpheus` python package. It also pulls down all the necessary Conda runtime dependencies for the core stages including [`mrc`](https://anaconda.org/nvidia/mrc) and [`libmrc`](https://anaconda.org/nvidia/libmrc).
+### Install additional `Pypi` dependencies
+Some of the stages in the core library require additional dependencies that are hosted on `Pypi`. These dependencies are included as a requirements file in the `morpheus` python package. The requirements files can be located and installed by running the following command:
+```bash
+python3 <<EOF
+import importlib.resources
+import subprocess
+requirements_file = importlib.resources.path("morpheus", "requirements_morpheus_core.txt")
+subprocess.call(f"pip install -r {requirements_file}".split())
+EOF
```
[the remark this answers is not in the document]
Comment (AnuradhaKaruppiah) [reply docs/source/conda_packages.md:59]: What do I know abt bash :) But it does look simpler.

**Comment by AnuradhaKaruppiah** [comment]
Thx for the review and tips @dagardner-nv

**Pull request merged**